tests/multi/bmp180.test.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright 2024 Khalil Estell
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <libhal-sensor/multi/bmp180.hpp>
+
+#include "../../src/multi/bmp180_internal.hpp"
```

Comment:
```suggestion
#include "multi/bmp180_internal.hpp"
```

You should only need to do this for the include.